academy/3-ibc/7-ibc-tooling.md

```diff
@@ -88,20 +71,19 @@ There are canonical channels for ICS-20. All other channels will have been creat
 
 </HighlightBox>
 
-When you go back to the overview, below the visual overview of the entire network you can find a list of the most active zones by IBC volume in USD:
+When you go to the `Zones`, you can find a list of the most active zones by IBC volume in USD:
 
 ![Most active zones](/academy/3-ibc/images/mostactivezones.png)
 
 The list includes very useful information, such as:
 
 * `IBC volume, $`: USD value of tokens successfully relayed via IBC transfer with pertinent volume in progress.
-* `IBC transfer`: number of successfully relayed IBC transfers with pertinent quantity in progress.
-* `IBC volume out, $`: USD value of tokens successfully transferred to other zones with pertinent volume in progress.
 * `IBC volume in, $`: USD value of tokens successfully received from other zones with pertinent volume in progress.
-* `IBC WAU`: number of zone's unique addresses initiated outward IBC transfer(s).
-* `IBC volume activity`: graph visualizing the activity.
+* `IBC volume out, $`: USD value of tokens successfully transferred to other zones with pertinent volume in progress.
+* `IBC transfer`: number of successfully relayed IBC transfers with pertinent quantity in progress.
+* `IBC DAU`: number of zone's unique addresses initiated outward IBC transfer(s).
```

Comment:
> number of zone's unique addresses initiated outward IBC transfer(s).

Is this to say "the number of unique addresses within the zone that have initiated outward IBC transfers"?

Reply:
Yes 👍